client/src/App.js

```diff
@@ -1,26 +1,17 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+// import React from 'react';
+// import { BrowserRouter as Router, Route } from 'react-router-dom';
+// import './App.css';
+// import GlobalStyles from 'Globalstyles.js';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
+// function App() {
+//   return (
+//     <GlobalStyles>
+//       <Router>
+//         <Route path="/" />
```

Comment:
I'd suggest changing this to "exact path"

Reply:
Yes thanks i saw it in your pull request :)